testing/go/types_test.go

```diff
@@ -497,16 +507,46 @@ var typesTests = []ScriptTest{
 		Name: "JSON type",
 		SetUpScript: []string{
 			"CREATE TABLE t_json (id INTEGER primary key, v1 JSON);",
-			"INSERT INTO t_json VALUES (1, '{\"key\": \"value\"}'), (2, '{\"num\": 42}');",
+			"INSERT INTO t_json VALUES (1, '{\"key\": \"value\"}'), (2, '{\"num\":42}');",
```

Comment:
Would be nice to see some tests involving sort ordering and equality in the face of e.g. different object key orderings. Not sure what postgres does here, would be nice to document.

Reply:
Will add these in a separate PR.